common/windows/delphi/tools/sentrytool/Makefile

```diff
@@ -4,9 +4,8 @@
 
 !include ..\..\Defines.mak
 
-build: dirs
+build:
     $(DELPHI_MSBUILD) sentrytool.dproj "/p:Platform=Win32"
-    $(COPY) $(WIN32_TARGET_PATH)\sentrytool.exe $(PROGRAM)\buildtools
```

Comment:
So, we're no longer copying this file?  Why were we copying it before, but not now?

Reply:
For now, I've decided to reference the tools in common/windows/ from their build folders (`WIN32_TARGET_PATH`, which is `bin/win32/debug` or `bin/win32/release` depending on the build mode) rather than copying them to another folder -- it's unnecessary for build-time components.